lib/Migration/Version19000Date20241029123147.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\GroupFolders\Migration;
+
+use Closure;
+use OCP\DB\ISchemaWrapper;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+class Version19000Date20241029123147 extends SimpleMigrationStep {
+	/**
+	 * @param IOutput $output
+	 * @param Closure(): ISchemaWrapper $schemaClosure
+	 * @param array $options
+	 * @return null|ISchemaWrapper
+	 */
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options): ?ISchemaWrapper {
+		/** @var ISchemaWrapper $schema */
+		$schema = $schemaClosure();
+
+		$table = $schema->getTable('group_folders');
+		if (!$table->hasIndex('group_folders_folders')) {
+			$table->addIndex(['folder_id', 'mount_point', 'quota', 'acl'], 'group_folders_folders');
```

Comment:
Including those 4 columns means that we need to have them in the request conditions. Are all requests following that format?

```sql
SELECT ...
WHERE
  folder_id = ... AND
  mount_point = ... AND
  quota = ... AND
  acl = ...
```

Reply:
Eh you're right, I confused it with the selected columns...